src/App.tsx

```diff
@@ -1,14 +1,76 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, {
+  useEffect,
+  useState,
+  useCallback,
+  useMemo,
+} from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { TodosFilter } from './types/TodosFilter';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [filter, setFilter] = useState(TodosFilter.ALL);
+  const [query, setQuery] = useState('');
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const loadTodos = useCallback(async () => {
+    try {
+      const todosFromServer = await getTodos();
+
+      setIsLoading(false);
+      setTodos(todosFromServer);
+    } catch (error) {
+      setIsError(true);
+    }
+  }, []);
+
+  useEffect(() => {
+    loadTodos();
+  }, [loadTodos]);
```

Comment:
move to the end of the code 
```suggestion
  useEffect(() => {
    loadTodos();
  }, []);
```